flink-end-to-end-tests/test-scripts/test_tpcds.sh

```diff
@@ -66,10 +66,10 @@ function run_test() {
         set_config_key "taskmanager.numberOfTaskSlots" "4"
     elif [ "${scheduler}" == "AdaptiveBatch" ]; then
```

Comment:
Currently, we recommend to use `execution.batch.adaptive.auto-parallelism.enabled` instead of  `jobmanager.scheduler`, so I recommend to change this.

Reply:
After offline discussion, this test is used for the function of the default scheduler and the correctness of the adaptive batch scheduler after all functions(auto-parallelism, speculative) are enabled. So there will not change to use `execution.batch.adaptive.auto-parallelism.enabled`.